src/batch.js

```diff
@@ -185,6 +184,7 @@ function validateBatchRequest(body, maxBatchSize) {
 
   // validate each body.requests[]
   const ids = body.requests.map(req => req.id)
+  const methods = ['GET', 'POST', 'PUT', 'PATCH', 'DELETE']
```

Comment:
I think @arturozv meant const as in declared once outside the function, instead of being declared on each validation run

Reply:
![Alt Text](https://media.giphy.com/media/fikcKja7O7MtzXzvQy/giphy.gif)